src/effects/native/echoeffect.cpp

```diff
@@ -32,7 +32,7 @@ EffectManifest EchoEffect::getManifest() {
     manifest.setShortName(QObject::tr("Echo"));
     manifest.setAuthor("The Mixxx Team");
     manifest.setVersion("1.0");
-    manifest.setDescription(QObject::tr("Stores the input signal in a temporary buffer"
+    manifest.setDescription(QObject::tr("Stores the input signal in a temporary buffer "
```

Comment:
Why we need a space here?

Reply:
Because the string continues on the next line

Reply:
This is a translators trap. Can we combine both strings? Or can we exclude the space  part from translations?

Reply:
Sure, we can combine the strings if you're okay with long lines of code.

Reply:
Are you sure the strings aren't put together on Transiflex? @esbrandt do you know?

Reply:
It is fine, and used in different places. Strings inside `tr()` are treated as paragraph. So the translator will see the whole sentence here. Transifex even support translators with consistency checks and markers for invisible elements like space and carriage return.

Reply:
Thank you for clarifying. I reformatted the code so these awkward line breaks are not necessary anyway.